src/main/java/esi/backend/controller/RentalController.java

```diff
@@ -24,28 +25,30 @@ public List<Rental> getAllRentals(@PathVariable UUID carId) {
         return rentalService.getAllRentals(carId);
     }
 
-    @RequestMapping("/cars/rentals/{id}")
-    public Optional<Rental> getRental(@PathVariable UUID id) {
-        return rentalService.getRental(id);
+    @RequestMapping("/cars/rentals/{rentalId}")
+    public Optional<Rental> getRental(@PathVariable UUID rentalId) {
+        return rentalService.getRental(rentalId);
```

Comment:
I think this and `getAllRentals` also need only manager authorization because otherwise, any user can check any rental with the customer details etc. The get methods for customers (`Get /customer/{customerId}/rentals/{rentalId}`) are yet to be added for customer access, where the customer ID has to be checked.

Reply:
Yes, no worries, I was just justifying the decision of why these must have manager authorization. Adding customer requests later is fine :).